src/Libraries/Microsoft.Extensions.AI.Evaluation.Reporting/CSharp/Microsoft.Extensions.AI.Evaluation.Reporting.csproj

```diff
@@ -33,59 +34,4 @@
     <InternalsVisibleToTest Include="Microsoft.Extensions.AI.Evaluation.Reporting.Tests" />
   </ItemGroup>
 
-  <!-- Build the report code before this, so it can be embedded.
```

Comment:
Similar target also exists in the Console project. Could you please move that one out too?

Reply:
I'm probably not going to have time to refactor this today. I'll do it as a follow-up task given the urgency to get the build updated.